pkg/workloads/cortex/client/setup.py

```diff
@@ -20,7 +20,7 @@
     description="",
     author="Cortex Labs",
     author_email="dev@cortexlabs.com",
-    install_requires=["dill==0.3.0", "requests==2.21.0"],
+    install_requires=["dill==0.3.0", "requests==2.21.0", "msgpack==0.6.1"],
```

Comment:
Can we, and should we, be more flexible with these versions since the user will be installing this in their environment?